cartoframes/examples.py

```diff
@@ -0,0 +1,5 @@
+from cartoframes import CartoContext
```

Comment:
We should add a simple module docstring here for documentation purposes.

Reply:
Ok. I'll wait for the decision on the approach for `exampleContext`.